example/web-node-sample/README.md

```diff
@@ -0,0 +1,20 @@
+# Web-node validation
+This sample code is to demonstrate that the node sdk can be used to execute Scalar DL binary request build by another application using any Scalar sdk.
+
+![](img/scalar_arch.png)
+
+In conclusion, the listContracts request sent by Web SDK can be executed by ScalarDLT invoked from node sdk if the
+`ClientProperties` provided by the Web SDK is appropriate. 
```

Comment:
I think it would be more appropriate to mention about the `ClientProperties` of the node-sdk in the `## Project Prerequisite` paragraph as it explains how to set up Scalar DL properly.